scripts/golangci.yml.in

```diff
@@ -179,3 +179,10 @@ linters:
     # TODO: enable wrapcheck linter
     # See: https://github.com/ceph/ceph-csi/pull/2268
     - wrapcheck
+    # TODO: enable linters added in golangci-lint 1.43
```

Comment:
have you created a github issue for this?

Reply:
will reuse the existing issue as I have just added an update in the PR description. once PR is merged I will add a link to disabled linters and comment on the issue for tracking.

Reply:
here is the updated comment https://github.com/ceph/ceph-csi/issues/2595#issuecomment-1018385617